docs/adr/0028-use-react-testing-library.md

```diff
@@ -0,0 +1,45 @@
+
+# *Use react-testing-library in place of Enzyme*
+
+**User Story:**  *[ES 809](https://jiraent.cms.gov/browse/ES-809)*
+
+  Currently, our frontend test coverage is abysmal. It's sitting right around 30%.
+  As a team, we've talked about being more intentional about writing unit
+  tests, but often times unit tests were taking longer than the feature
+  themselves. We realized that writing tests with [Enzyme](https://enzymejs.github.io/enzyme/)
+  was making testing harder and confusing.
+
+[Enzyme](https://enzymejs.github.io/enzyme/) was released in 2016 by AirBnb.
+At the time, it was the gold standard for testing in the React ecosystem.
+However by 2019, the React API has changed dramatically and support for
+[Enzyme](https://enzymejs.github.io/enzyme/) has decreased with  [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/)
+picking up the market share.
+
+**Sources**
+[Time to say goodbye - Enzyme.js](https://www.piotrstaniow.pl/goodbye-enzyme)
+
+## Considered Alternatives
```

Comment:
I'd add "staying with Enzyme" as one of the alternatives, and then add the pros and cons at the category at the bottom